test/lib/formValidator.test.ts

```diff
@@ -0,0 +1,217 @@
+import { FormValidator } from "../../src/lib/formValidator";
+
+describe("FormValidator", () => {
+  describe("validate", () => {
+    it("when given a form data object with a valid field, returns a 'valid' response", () => {
```

Comment:
Each of these test cases are fairly long in length. You think it might be possible to reduce duplication between tests with a helper function?

Reply:
Oooh yes that's a good one.  I've extracted the creating of the validator function mocks to two helper methods (`valid` and `invalid`).  

I could slim down further by removing the `value` pass-through on `FieldValidator` as we discussed?